content/20.treatments.md

```diff
@@ -105,6 +105,49 @@ Summarize the state of the antiviral approach.
+In coronaviruses specifically, in vitro evidence demonstrates that the combination of zinc (Zn2+) and zinc ionophores (pyrithione) can interrupt the replication mechanisms of SARS-CoV-GFP and a variety of other RNA viruses [@doi:10.1371/journal.ppat.1001176; @doi:10.1016/j.antiviral.2014.12.015]. 
+While overall there is encouraging data for zinc supplementation against the common cold, there is currently limited evidence to suggest zinc supplementation has any beneficial effects against the current novel COVID-19, thus further research is imperative.
+
+Vitamins B, C, D, and E have also been suggested as potential nutrient supplement interventions for COVID-19 [@doi:10.1002/jmv.25707; @doi:10.20944/preprints202003.0347.v1].
+Despite that it is well documented that regular vitamin C supplementation may reduce the duration of the common cold, vitamin C (> 200 mg) fails to reduce the incidence of  [@doi:10.1002/14651858.CD000980.pub4.].
+Vitamin C also reduces the susceptibility of patients to lower respiratory tract infections such as pneumonia [@doi:10.1097/00006454-199709000-00003].
+A recent meta-analysis has demonstrated in twelve trials that vitamin C supplementation reduced the length of stay of patients in intensive care units (ICUs) by 7.8% (95% CI: 4.2% to 11.2%; p = 0.00003).
+Furthermore, high doses (1-3 g/day) significantly reduced the length of an ICU stay by 8.6% in six trials (p = 0.003).
+Vitamin C also shortened the duration of mechanical ventilation by 18.2% in three trials in which patients required intervention for over 24 hours (95% CI 7.7% to 27%; p = 0.001)  [@doi:10.3390/nu11040708]. 
+Despite these findings, the CITRUS ALI study failed to show a benefit of a 96-hour infusion of vitamin C to treat ARDS, which is a severe complication of COVID-19 infection [@doi:10.1001/jama.2019.11825].
+There is anecdotal evidence from China and the United States that vitamin C is currently being used for the treatment of COVID-19, but further controlled research is required.    
+
+In terms of other dietary supplements, vitamin D can modulate the adaptive and innate immune system.
+Vitamin D deficiency is associated with an increased susceptibility to infection [@doi:10.231/JIM.0b013e31821b8755].
+A recent preprint postulated that an individual’s vitamin D status may significantly affect one’s risk of developing COVID-19. 
+This hypothesis was derived from the fact that the current pandemic occurred in winter in Wuhan China when 25-hydroxyvitamin D concentrations are at their lowest due to lack of sunlight, whereas in the Southern Hemisphere, where it was nearing the end of the summer, the number of cases was low at that time coinciding with higher 25-hydroxyvitamin D concentrations  [@doi:10.20944/preprints202003.0235.v2].
+However, this has yet to be determined as it is not yet even known whether COVID-19 is seasonal. 
```

Comment:
I only skimmed this preprint, but I was wondering if it might be worth mentioning that there are many factors that correlate with season (and old age), one of which is population-scale vitamin D levels, so the vitamin D --> infection risk causal relationship that study is putting forward is one explanation among many possibilities for why the correlation might be observed. It also sounds like many of the background studies showed that having a deficiency vs not having a deficiency may be the important factor? I have to admit, I'm a bit skeptical of this preprint just based on their conflicts of interest, so apologies if I'm missing something!

Reply:
Good suggestions, I had thought to expand this further. The only reason I didn't is I thought I had gone too far! But I think you are right, I will make an edit on that, feel free to edit